src/components/Header/HeaderLinks.js

```diff
@@ -0,0 +1,22 @@
+import Link from './Link';
+function HeaderLinks(props) {
+  const { headerLinksConfig } = props;
+
+  return (
+    <>
+      <nav className='navbar'>
+        <ul>
+          {headerLinksConfig.map((item, index) => {
```

Comment:
Move this iteration fn to a helper one (either in a separate file or above the functional component in the same file)